ouroboros-network/src/Ouroboros/Network/PeerSelection/PeerMetric.hs

```diff
@@ -163,5 +163,24 @@ fetchyness = Pq.fold' count Map.empty
         fn Nothing = Just $ fromIntegral bytes
         fn (Just oldBytes) = Just $! oldBytes + fromIntegral bytes
 
+-- Returns a Map which counts the number of times a given peer
+-- was the first we downloaded a block from.
+fetchyness'
```

Comment:
Could you rename  `fetchyness'` maybe as `fetchynessBlockCount` and also rename `fetchyness` as `fetchynessBytes`.  Or something like that.